assets/app/views/builds.html

```diff
@@ -100,7 +100,7 @@
                         <span ng-if="build.spec.source.type == 'None'">
                           <i>none</i>
                         </span>
-                        <span ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
+                        <span class="word-break" ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
```

Comment:
@spadgett Actually we didn't switch the builds table over to table-layout:fixed so that means it would need to be class="word-break-all" so that FF and IE will break long git url strings.